.github/workflows/main.yml

```diff
@@ -17,9 +17,6 @@ jobs:
     - uses: gozala/typescript-error-reporter-action@v1.0.8
     - run: npx aegir build
     - run: npx aegir dep-check
-    - uses: ipfs/aegir/actions/bundle-size@master
```

Comment:
this needs to be added again once aegir ships it as I could not make it use a branch

Reply:
I've restored this but now CI is failing on this step with:

```
Error: Cannot find module '/github/workspace/index.js'
```

Reply:
oh damn, I will look into it